make-intranet-public/README.md

```diff
@@ -0,0 +1,63 @@
+# Making parts of the intranet public
+
+## Status quo
+
+We currently have an internal `hugo` page which we use for internal documentation.
+This page is called `intranet` and contains all kinds of documentation around our operational work as well as security and development.
+The source code for our `intranet` is in a private GitHub repository which contains other security and business relevant data.
+
+## Goal
+
+The non-security relevant parts of the intranet should be public.
+Making them public enables us to collaborate with other companies more easily.
+It additionally allows us to enable outside contributors to work on projects more independently.
+
```

Comment:
I'd add that it will also work towards branding and especially employer branding.